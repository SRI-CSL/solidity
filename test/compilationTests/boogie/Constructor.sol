pragma solidity ^0.4.17;

contract Constr {
    uint value;

    constructor(uint init) public {
        value = init;
    }

    function doSomething(uint param) public returns (uint) {
        value += param;
        return value;
    }
}