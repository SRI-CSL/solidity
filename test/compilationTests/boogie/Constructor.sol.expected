Boogie program verifier finished with 3 verified